test/Microsoft.Extensions.DependencyInjection.Ordered.Tests/ServiceCollectionOrderedServiceExtensionsTest.cs

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.DependencyInjection.Ordered;
+using Microsoft.Extensions.DependencyInjection.Specification.Fakes;
+using Xunit;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public class ServiceCollectionOrderedServiceExtensionsTest
```

Comment:
It's free tests for container authors in case they decides to re-implement `IOrdered`.